apps/condo/domains/miniapp/schema/DescriptionBlock.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Generated by `createschema miniapp.DescriptionBlock 'description:Text; image:File;'`
+ */
+
+const { Text, Relationship, Integer } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/miniapp/access/DescriptionBlock')
+const { APP_IMAGE_FIELD } = require('@condo/domains/miniapp/schema/fields/integration')
+const { WRONG_AMOUNT_OF_BLOCK_ERROR } = require('@condo/domains/miniapp/constants')
+const get = require('lodash/get')
+
+const RELATION_FIELDS = ['billingIntegration', 'acquiringIntegration']
+
+const ONE_OF_ALL_MESSAGE = `Note that one of the following fields are required: [${RELATION_FIELDS.join(', ')}]`
+
+
+const DescriptionBlock = new GQLListSchema('DescriptionBlock', {
```

Comment:
Why it’s not just a field inside the app?

Reply:
Since app have multiple description blocks containing title, description and image....grouping them here is good way to order them properly instead of have 3 lists on app (2x lists of Text and 1x of File)

Reply:
This model was deleted